testing/stargazer/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Alexander Georgievskiy <galeksandrp@gmail.com>
+# Maintainer:
+pkgname=stargazer
+pkgname2=stg
```

Comment:
When you need to define custom global variable, please prefix it with underscore (to avoid potential conflicts with future abuild variables). Common name for this variable is `_pkgname`.